### ISSUE TITLE
Bamberg - Athen does not work

### ISSUE THREAD

**Issue by MRLB** [hubsif/kodi-tk_bbl#11]
The match between Bamberg and Athen is not working yet.

Both Eurocup matches (Ulm and Berlin) Work.

Telekom-problem or a issue with the addon?

**Comment by MRLB** [comment]
Logfile:
`20:53:17 T:1784   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--

```
                                         - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                        Error Type: <class 'xml.etree.ElementTree.ParseError'>
                                        Error Contents: syntax error: line 1, column 0
                                        Traceback (most recent call last):

                                          File "C:\Users\Benutzer\AppData\Roaming\Kodi\addons\plugin.video.tk_bbl\default.py", line 215, in <module>

                                            xmlroot = ET.ElementTree(ET.fromstring(response))
                                          File "C:\Program Files (x86)\Kodi\system\python\Lib\xml\etree\ElementTree.py", line 1300, in XML

                                            parser.feed(text)
                                          File "C:\Program Files (x86)\Kodi\system\python\Lib\xml\etree\ElementTree.py", line 1642, in feed

                                            self._raiseerror(v)
                                          File "C:\Program Files (x86)\Kodi\system\python\Lib\xml\etree\ElementTree.py", line 1506, in _raiseerror

                                            raise err
                                        ParseError: syntax error: line 1, column 0
                                        -->End of Python script error report<--`
```

**Comment by hubsif** [comment]
Just checked. Though I'm getting a different error than you, it's again a telekom issue (invalid video id).

I hope I'll find some time soon to release v2 based on the new api. And I will try to improve error handling, so that the addon doesn't crash but display a message in such cases.

**Comment by hubsif** [comment]
Please have a look at the releases page. I created an alpha version using the new API. That way you might be able to still watch the affected streams.

**Comment by MRLB** [comment]
Nice. Thank you. I have tried some videos today and everything have worked fine. But maybe it could be possible to sort the Gamereports and Replays like at the Eishockey addon after matchday.

I found two 'issues'. In line 128 there is a link to Telekom Eishockey. Maybe it does not matter but maybe it could be a problem during live matches.

Also all live matches are scheduled 15 minutes too early. For example tomorrow the stream of Bamberg - Barcelona will start at 8:45pm. The addon means 8:30pm. The BBL match is 15 minutes to early too.

Thank you for your fast changes. I will test the addon tomorrow during the two live matches.
